go/test/endtoend/cluster/topo_process.go

```diff
@@ -332,6 +330,10 @@ func (topo *TopoProcess) ManageTopoDir(command string, directory string) (err er
 		}
 		return err
 	} else if command == "rmdir" {
+		if *topoFlavor == "etcd2" { // Use etcdctl as the gRPC gateway
+			cmd := exec.Command("etcdctl", "del", "--prefix", directory)
+			return cmd.Run()
+		}
```

Comment:
Can we do something like https://github.com/etcd-io/etcd/issues/11914#issuecomment-631078085 instead here so we don't have to invoke a subcommand?

Also I think if the comment for `ManageTopoDir` is correct, we only need one implementation here as well?

Reply:
Yes, we could (I am aware of that method which takes base64 encoded parameters but it's awkward in a variety of ways). If this was not endtoend test specific then I would go that route. But sub-commands are not unusual in the e2e tests, this is the more canonical way to do it, and it's rarely done. So I prefer this method.

Reply:
Even better... I'm going to switch to using the go client package: https://pkg.go.dev/go.etcd.io/etcd/client/v3#section-readme

Reply:
Great!